src/experiments.jl

```diff
@@ -51,7 +51,9 @@ function check_params(edit_params_f, savepath, data, parameters)
 	if ~isdir(savepath)
 		return true
 	end
-	saved_params = map(x -> DrWatson.parse_savename(x)[2], readdir(savepath))
+	# filter out duplicates created by tagsave
+	fs = filter(x->!(occursin("_#", x)), readdir(savepath))
```

Comment:
What exactly was the problem here? Are there some temporary files that DrWatson creates containing `"_#"`?

Reply:
Yes, if you use tagsave with safesave feature and the new saved file is named the same as an existing file, it appends "_#1" or some other number. But then parse_savename cannot parse this file and everything crashes :-)